release-notes/9.0/linux-packages.json

```
@@ -0,0 +1,408 @@
+{
+    "channel-version": "9.0",
+    "last-updated": "2024-07-01",
+    "packages": [
+        {
+            "id": "libc",
+            "name": "C Library",
+            "references": [
+                "https://www.gnu.org/software/libc/libc.html",
+                "https://musl.libc.org/"
+            ],
+            "required": [
+                "CoreCLR",
+                "Native AOT"
+            ]
+        },
+        {
+            "id": "libgcc",
+            "name": "GCC low-level runtime library",
+            "references": [
+                "https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html"
+            ],
+            "required": [
+                "CoreCLR",
+                "Native AOT"
+            ]
+        },
+        {
+            "id": "ca-certificates",
+            "name": "CA Certificates",
+            "references": [
+                "https://www.redhat.com/sysadmin/ca-certificates-cli"
+            ],
+            "required": [
+                "HTTPS"
+            ]
+        },
+        {
+            "id": "libssl",
```

Thoughts on making it more explicit that this is OpenSSL, not another SSL library?

```suggestion
            "id": "openssl",
```